ansible/vars/topo_t2.yml

```diff
@@ -0,0 +1,1867 @@
+      vm_offset: 55
+    ARISTA25T1:
+      vlans:
+        - 1.24@56
+      vm_offset: 56
+    ARISTA26T1:
+      vlans:
+        - 1.25@57
+      vm_offset: 57
+    ARISTA27T1:
+      vlans:
+        - 1.26@58
+      vm_offset: 58
+    ARISTA28T1:
+      vlans:
+        - 1.27@59
+      vm_offset: 59
+    ARISTA29T1:
+      vlans:
+        - 1.28@60
+      vm_offset: 60
+    ARISTA30T1:
+      vlans:
+        - 1.29@61
+      vm_offset: 61
+    ARISTA31T1:
+      vlans:
+        - 1.30@62
+      vm_offset: 62
+    ARISTA32T1:
+      vlans:
+        - 1.31@63
+      vm_offset: 63
+    ARISTA33T1:
+      vlans:
+        - 2.0@64
+        - 2.1@65
+      vm_offset: 64
+    ARISTA35T1:
+      vlans:
+        - 2.2@66
+        - 2.3@67
+      vm_offset: 66
+    ARISTA37T1:
+      vlans:
+        - 2.4@68
+        - 2.5@69
+      vm_offset: 68
+    ARISTA39T1:
+      vlans:
+        - 2.6@70
+        - 2.7@71
+      vm_offset: 70
+    ARISTA41T1:
+      vlans:
+        - 2.8@72
+        - 2.9@73
+      vm_offset: 72
+    ARISTA43T1:
+      vlans:
+        - 2.10@74
+        - 2.11@75
+      vm_offset: 74
+    ARISTA45T1:
+      vlans:
+        - 2.12@76
+        - 2.13@77
+      vm_offset: 76
+    ARISTA47T1:
+      vlans:
+        - 2.14@78
+        - 2.15@79
+      vm_offset: 78
+    ARISTA49T1:
+      vlans:
+        - 2.16@80
+      vm_offset: 80
+    ARISTA50T1:
+      vlans:
+        - 2.17@81
+      vm_offset: 81
+    ARISTA51T1:
+      vlans:
+        - 2.18@82
+      vm_offset: 82
+    ARISTA52T1:
+      vlans:
+        - 2.19@83
+      vm_offset: 83
+    ARISTA53T1:
+      vlans:
+        - 2.20@84
+      vm_offset: 84
+    ARISTA54T1:
+      vlans:
+        - 2.21@85
+      vm_offset: 85
+    ARISTA55T1:
+      vlans:
+        - 2.22@86
+      vm_offset: 86
+    ARISTA56T1:
+      vlans:
+        - 2.23@87
+      vm_offset: 87
+    ARISTA57T1:
+      vlans:
+        - 2.24@88
+      vm_offset: 88
+    ARISTA58T1:
+      vlans:
+        - 2.25@89
+      vm_offset: 89
+    ARISTA59T1:
+      vlans:
+        - 2.26@90
+      vm_offset: 90
+    ARISTA60T1:
+      vlans:
+        - 2.27@91
+      vm_offset: 91
+    ARISTA61T1:
+      vlans:
+        - 2.28@92
+      vm_offset: 92
+    ARISTA62T1:
+      vlans:
+        - 2.29@93
+      vm_offset: 93
+    ARISTA63T1:
+      vlans:
+        - 2.30@94
+      vm_offset: 94
+    ARISTA64T1:
+      vlans:
+        - 2.31@95
+      vm_offset: 95
+
+
+configuration_properties:
+  common:
+    dut_asn: 65100
+    dut_type: LeafRouter
+    nhipv4: 10.10.246.254
+    nhipv6: FC0A::FF
+  spine:
+    swrole: spine
+
+configuration:
+  ARISTA01T3:
+    properties:
+      - common
+      - spine
+    bgp:
+      asn: 65200
+      peers:
+        65100:
+          - 20.0.0.0
+          - FC00::1
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.1/32
+        ipv6: 2064:100::1/128
+      Ethernet1:
+        lacp: 1
+      Ethernet2:
+        lacp: 1
+      Port-Channel1:
+        ipv4: 20.0.0.1/31
```

Comment:
Why assign PortChannel IP address 20.0.0.x? In other topologies, IP address 10.0.0.x is used.

Reply:
Currently in our SONiC Chassis, the subnet 10.0.0.0/24 is used for backplane (midplane) connectivity between the cards (used for synchronization of VOQ related databases as well). Thus the default address scheme conflicts with the midplane address.  Will ask developers if the midplane address can be configurable.

Reply:
Committed the changes